pkg/ingress/apisix/tls.go

```diff
@@ -0,0 +1,68 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package apisix
+
+import (
+	ingressConf "github.com/api7/ingress-controller/pkg/kube"
+	ingress "github.com/gxthrj/apisix-ingress-types/pkg/apis/config/v1"
+	apisix "github.com/gxthrj/apisix-types/pkg/apis/apisix/v1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ApisixTls = "ApisixTls"
```

Comment:
We should change all `Tls` to `TLS`.

Reply:
We should not change CRD name, for that it will be format as `apisixtLS`